services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -86,3 +90,19 @@ func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
 		klog.Errorf("failed write data to response writer: %v", err)
 	}
 }
+
+func newClient() (client.Client, error) {
```

Comment:
this is a handler and possibly creation of client shouldn't be here.

Reply:
Ack,  removed the code in handler.go and as suggested called from the main.